### PR TITLE
Update StandardActions.java : typo corrected

### PR DESCRIPTION
**Pull request by mlep**
Just the correction of a typo (double dots)

### Mandatory checks

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Review by Siedlerchr — DISMISSED** [review]

File: src/main/java/org/jabref/gui/actions/StandardActions.java [hunk 1]
```diff
@@ -94,7 +94,7 @@ public enum StandardActions implements Action {
     PARSE_LATEX(Localization.lang("Search for citations in LaTeX files..."), IconTheme.JabRefIcons.LATEX_CITATIONS),
     NEW_SUB_LIBRARY_FROM_AUX(Localization.lang("New sublibrary based on AUX file") + "...", Localization.lang("New BibTeX sublibrary") + Localization.lang("This feature generates a new library based on which entries are needed in an existing LaTeX document."), IconTheme.JabRefIcons.NEW),
     NEW_LIBRARY_FROM_PDF_ONLINE(Localization.lang("New library based on references in PDF file... (online)"), Localization.lang("This feature generates a new library based on the list of references in a PDF file. Thereby, it uses GROBID's functionality."), IconTheme.JabRefIcons.NEW),
-    NEW_LIBRARY_FROM_PDF_OFFLINE(Localization.lang("New library based on references in PDF file... (offline)"), Localization.lang("This feature generates a new library based on the list of references in a PDF file. Thereby, it uses JabRef's build-in functionality.."), IconTheme.JabRefIcons.NEW),
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/actions/StandardActions.java:97]: must be correct in the en.properties as well

Comment (mlep) [reply src/main/java/org/jabref/gui/actions/StandardActions.java:97]: Should be corrected. Thanks for the review.

File: src/main/resources/l10n/JabRef_en.properties [hunk 2]
```diff
@@ -895,7 +895,7 @@ nested\ AUX\ files=nested AUX files
 New\ library\ based\ on\ references\ in\ PDF\ file...\ (offline)=New library based on references in PDF file... (offline)
 New\ library\ based\ on\ references\ in\ PDF\ file...\ (online)=New library based on references in PDF file... (online)
 This\ feature\ generates\ a\ new\ library\ based\ on\ the\ list\ of\ references\ in\ a\ PDF\ file.\ Thereby,\ it\ uses\ GROBID's\ functionality.=This feature generates a new library based on the list of references in a PDF file. Thereby, it uses GROBID's functionality.
-This\ feature\ generates\ a\ new\ library\ based\ on\ the\ list\ of\ references\ in\ a\ PDF\ file.\ Thereby,\ it\ uses\ JabRef's\ build-in\ functionality..=This feature generates a new library based on the list of references in a PDF file. Thereby, it uses JabRef's build-in functionality..
+This\ feature\ generates\ a\ new\ library\ based\ on\ the\ list\ of\ references\ in\ a\ PDF\ file.\ Thereby,\ it\ uses\ JabRef's\ build-in\ functionality.=This feature generates a new library based on the list of references in a PDF file. Thereby, it uses JabRef's build-in functionality.
```
Comment (Siedlerchr) [inline src/main/resources/l10n/JabRef_en.properties:898]: Here it must be now built-in as well

**Comment by Siedlerchr** [comment]
I wonder why the tests didn't catch this..

**Comment by mlep** [comment]
> I wonder why the tests didn't catch this..

I wonder why my brain didn't catch this...

**Comment by koppor** [comment]
> I wonder why the tests didn't catch this..

Please unzip for me so that I can have a look :)

**Comment by Siedlerchr** [comment]
> > I wonder why the tests didn't catch this..
> 
> Please unzip for me so that I can have a look :)

The L10n tests should have failed earlier after the first commit

**Comment by koppor** [comment]
> > Please unzip for me so that I can have a look :)
> The L10n tests should have failed earlier after the first commit

Deep link: [`856ec41` (#11560)](https://github.com/JabRef/jabref/pull/11560/commits/856ec41472e73ef231d1029675f2bb91e47b3173)

**Review by koppor — APPROVED** [review]

**Pull request merged**